demos/tests/cases.py

```diff
@@ -705,6 +705,15 @@ def single_option_cases(key, *args):
         ],
     )),
 
+    PythonDemo(name='background_subtraction_demo', device_keys=['-d'], test_cases=combine_cases(
```

Comment:
please comment out test case for now, we will uncomment this, once new model package will became available for download

Reply:
Done.